Add License

fifieldt opened the pull request:
* Code contributions are licensed under the Apache 2.0 License.
* Documentation contributions are licensed under the CC-BY-SA-NC license.

Add LICENSE.md file that includes both licenses.

Fixes #4

erickhun commented:
LGTM @fifieldt ! also wondering if we should link it somehow in the about page to make things clear?

fifieldt commented:
agreed, I will amend the pull request.

fifieldt commented:
This needs an update, to make it explicit which bits are licensed under which license.

fifieldt commented:
Hi all,

I have updated the proposed changes with some more text in the license file and added information to the about page. Please take a look at the licenses and consider them carefully.

You might want to refer to https://choosealicense.com/licenses/ when thinking about Apache 2, and https://creativecommons.org/licenses/by-nc-sa/4.0/ for the creative commons.

fifieldt commented:
To date we have received acknowledgement of the license from: Alvin, Colum, David, Eric, Ian, Jonathan and myself. That's all of our contributors to date, so this patch is now ready for merge.
